test/data_processing/test_nodes.py

```diff
@@ -301,3 +302,29 @@ def test_probability_balanced(self):
         data = {"1": 512, "0": 512}
         processed_data = node(data=np.asarray([data]))
         self.assertAlmostEqual(unp.nominal_values(processed_data), 0.5)
+
+
+class TestRestless(QiskitExperimentsTestCase):
+    """Test the restless measurements node."""
+
+    def test_restless_classify(self):
```

Comment:
To be on the safe side can you also add a test for the case where the bitstring has length 1?

Reply:
See 546412a.